README.md

```diff
@@ -300,6 +300,8 @@ const [isStoryViewVisible, setIsStoryViewShow] = useState(false);
 
 <StoryContainer
   visible={isStoryViewVisible}
+  extended={true}
+  // extended enables play & pause feature on single story view default is "false"
```

Comment:
@sagar-simformsolutions We should revise this comment as follows

`  // extended enables play & pause feature on StoryContainer default is "false", MultiStory
`

Reply:
Even default should be true because we want to support that feature by default